server/routerlicious/.changeset/witty-years-exist.md

```diff
@@ -0,0 +1,13 @@
+---
+"@fluidframework/server-kafka-orderer": major
+"@fluidframework/server-lambdas": major
+"@fluidframework/server-memory-orderer": major
+"@fluidframework/server-routerlicious-base": major
+"@fluidframework/server-services-core": major
+"@fluidframework/server-services-ordering-kafkanode": major
+"@fluidframework/server-test-utils": major
```

Comment:
Does the breaking change apply to all of these packages? What is the "primary" package?

Reply:
All of these packages have breakign type changes for their exports except for `server-lambdas`. I can remove server-lambdas from this to reflect that

Reply:
The "primary" package(s) are:
1. services-core: exports all the interface definitions mentioned here
2. kafka-orderer: exports the IOrdererConnection implementation that needs to expose `.off` to call `.off` on its internal IProducer dependency

The rest all have exported implementations of those 2 interfaces, or exports that depend on those interfaces

Reply:
Please add three changesets total - one for services-core, one for kafka-orderer, and one for the others. Mark server-lambdas as minor in the third changeset (may need to do this manually). They can all be added in this PR.

Reply:
Went a slightly different route, because kafka-orderer fits in with "the others", while server-lambdas has the actual functionality change. Made 1 for services-core interface changes (major), 1 for server-lambdas functionality (minor), and 1 for others' type changes (major)